source/features/wait-for-build.tsx

```diff
@@ -32,9 +32,14 @@ function showCheckboxIfNecessary(): void {
 	const checkbox = getCheckbox();
 	const isNecessary = prCiStatus.get() === prCiStatus.PENDING;
 	if (!checkbox && isNecessary) {
-		const container = select('.commit-form-actions .select-menu');
-		if (container) {
-			container.append(generateCheckbox());
+		const containers = select.all('.commit-form-actions .select-menu');
+		for (const container of containers) {
+			const container_form = container.closest('form');
+			// Only show checkbox for merge form, not for auto merge form
+			if (container_form?.classList.contains('js-merge-form')) {
+				container.append(generateCheckbox());
+				break;
+			}
```

Comment:
It looks to me this code is just looking for `.js-merge-form`. Is this equivalent?
```suggestion
		for (const container of select.all('.js-merge-form')) {
			select('.select-menu', container)!.append(generateCheckbox());
```

Reply:
The only difference would be that this could possibly find multiple forms with `.js-merge-form` class and add the checkbox to all of them but there's only one present on GH's site so I guess that's not really a concern. But we could alternatively do this instead:
```suggestion
		const container = select('.js-merge-form');
		if (container)
			select('.select-menu', container)!.append(generateCheckbox());
```

Reply:
If it works in every case, go for it!

Also that's the same as 
```suggestion
		select('.js-merge-form .select-menu')?.append(generateCheckbox());
```

😄